frontend/src/Keyword/Keyword.module.css

```diff
@@ -1,30 +1,29 @@
-.DeleteKeyword {
+.keyword {
   background: hsl(0, 3%, 93%);
-  border-radius: 16px;
-  padding: 0.4em 0.6em 0.4em 0.8em;
-  margin: 0.5em;
-  margin-right: 1em;
+  border-radius: 8px;
+  padding: 0.6em 0.8em 0.6em 0.8em;
   display: flex;
   width: fit-content;
   align-items: center;
   height: 24px;
 }
 
+.DeleteKeyword {
+  border-radius: 16px;
+  padding: 0.4em 0.6em 0.4em 0.8em;
```

Comment:
⚠️ action required: padding and border radius seems inconsistent between types? is this because of the icon?